e2etests/tests/unnumbered.go

```diff
@@ -0,0 +1,285 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package tests
+
+import (
+	"fmt"
+	"net/netip"
+	"os"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	frrk8sv1beta1 "github.com/metallb/frr-k8s/api/v1beta1"
+	"github.com/metallb/frrk8stests/pkg/config"
+	"github.com/metallb/frrk8stests/pkg/dump"
+	"github.com/metallb/frrk8stests/pkg/infra"
+	"github.com/metallb/frrk8stests/pkg/k8s"
+	"github.com/metallb/frrk8stests/pkg/k8sclient"
+	"github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"go.universe.tf/e2etest/pkg/container"
+	"go.universe.tf/e2etest/pkg/executor"
+	"go.universe.tf/e2etest/pkg/frr"
+	"go.universe.tf/e2etest/pkg/netdev"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+
+	frrconfig "go.universe.tf/e2etest/pkg/frr/config"
+	frrcontainer "go.universe.tf/e2etest/pkg/frr/container"
+)
+
+var FRRImage string
+
+// The Unnumbered test can run on IPv4, IPv6 or Dual because it needs only the
+// IPv6 LLA addresses in the interface, not full IPv6 stack.
+var _ = ginkgo.Describe("Unnumbered", func() {
+	var (
+		nodeWithP2PConnection corev1.Node
+		externalP2PContainer  *frrcontainer.FRR
+		cs                    clientset.Interface
+		updater               *config.Updater
+		prefixSend            = []string{"5.5.5.0/24", "5555::/64"}
+	)
+
+	ginkgo.BeforeEach(func() {
+		if _, found := os.LookupEnv("RUN_FRR_CONTAINER_ON_HOST_NETWORK"); found {
```

Comment:
let's leave this to the caller. I'd rather have the runner willingly opt out instead of having discovery logic in the tests themselves

Reply:
done